internal/usage/usage.go

```diff
@@ -254,6 +254,7 @@ const (
 	SkipSampleData                  = "Don't load sample data into your Atlas Cluster."
 	ContainerRegion                 = "Atlas region where the container resides."
 	ContainerRegions                = "List of Atlas regions where the container resides."
+	Default                         = "Run the Quickstart command with all the auto generated values to deploy and access an Atlas cluster."
```

Comment:
[nit] just default may be too generic and prone to be badly reused 
```suggestion
	QuickstartDefault                         = "Run the Quickstart command with all the auto generated values to deploy and access an Atlas cluster."
```